src/stores/LayoutStore.js

```diff
@@ -0,0 +1,15 @@
+import { observable } from 'mobx';
+import { Bind } from 'lodash-decorators';
+
+class LayoutStore {
+
+  @observable docked = false;
```

Comment:
I would use some better name, this store may be used later for other things. Like `drawerDocked` or `menuDocked`